public/lesswrong-editor/src/ckeditor5-mention/src/mentionui.js

```diff
@@ -730,7 +730,8 @@ function getLastValidMarkerInText( feedsWithPattern, text ) {
 // @param {Number} minimumCharacters
 // @returns {RegExp}
 export function createRegExp( marker, minimumCharacters ) {
-  const numberOfCharacters = minimumCharacters == 0 ? '*' : `{${ minimumCharacters },}`;
+	// End after 20 characters, since it's horribly annoying otherwise.
+  const numberOfCharacters = minimumCharacters == 0 ? '*' : `{${ minimumCharacters },20}`;
```

Comment:
```suggestion
  const numberOfCharacters = minimumCharacters === 0 ? '*' : `{${ minimumCharacters },20}`;
```

Reply:
Why is it that minimum being 0 removes the max?

Reply:
Mostly an oversight - in practice both of our feeds have `minimumCharacters: 1`, but I agree that should be changed